### PR TITLE
Don't use runInSeparateProcess

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
There is not need in running a test in a separate process ....

## Motivation and Context
Clean code, better unit test execution

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
Hmm ok 👍

File: apps/dav/tests/unit/Connector/Sabre/FileTest.php [hunk 1]
```diff
@@ -351,7 +355,7 @@ public function testPutSingleFile() {
 	 * @return boolean true if negative mtime is supported
 	 */
 	private function supportsNegativeMtime() {
-		return (getenv("PRIMARY_STORAGE_CONFIG") !== "swift");
+		return !(getenv("RUN_OBJECTSTORE_TESTS") !== false);
```
Comment (phil-davis) [inline apps/dav/tests/unit/Connector/Sabre/FileTest.php]: Is this clear as mud to you?
having the 2 "!" - I guess the construction ``!== false`` is a way of saying "if true" -> "if RUN_OBJECTSTORE_TESTS" and then the "!" at the front negates that.

but it messes my brain a bit

Comment (DeepDiver1975) [inline apps/dav/tests/unit/Connector/Sabre/FileTest.php]: > but it messes my brain a bit

me too .... I need to understand the initial logic as well ... see rocket chat ....

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@DeepDiver1975 please backport

**Comment by DeepDiver1975** [comment]
Will do asap

**Pull request merged**